Customizing/global/plugins/Services/Cron/CronHook/UserOrguImport/classes/IliasGlobalRoleManagement.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace CaT\IliasUserOrguImport;
+
+/**
+ * Manipulate ilias-side role assignments. Wrapper around rbac.
+ */
+class IliasGlobalRoleManagement
```

Comment:
Why isn't there an interface implemented here? This is could be exactly that `how do I test stuff using ILIAS`-pattern I showed several times. Except it isn't.